views/files/file-viewer.hbs

```diff
@@ -1,16 +1,16 @@
 <section class="file-viewer">
-    <a href="#" class="lightbox videostop" id="file-view">
+    <a href="#" class="lightbox videostop" id="file-view" role="button" aria-label="{{$t "files.label.backToFiles"}}">
       <video id="my-video" class="video-js vjs-default-skin videoclick" autoplay controls preload="auto" data-setup='{"fluid": true}'></video>
 		  <script src="/vendor-optimized/video.js/dist/video.min.js" nonce="{{nonceValue}}"></script>
 		  <script src="/vendor-optimized/video.js/dist/lang/{{ @root.userLanguage }}.js" nonce="{{nonceValue}}"></script>
-      <img id="picture">
+      <img id="picture" alt="">
```

Comment:
can we add a meaningful alt here or use role="presentation"? :)

Reply:
A (more or less) meaningful alt is added when the img is loaded (see files.js changes)
Will that do? :)

Reply:
Why don't we add it directly here? It looks like it is just an i18n key.

Reply:
this alt might be a little unnecessary anyways: When clicking on an image in your files it is displayed here. This alt is loaded at that moment and reads something like this: "The Image picture.png"